src/ethereum/prague/vm/eoa_delegation.py

```diff
@@ -144,6 +165,9 @@ def set_delegation(message: Message, env: Environment) -> U256:
         if auth.chain_id not in (env.chain_id, U64(0)):
             continue
 
+        if auth.nonce >= 2**64 - 1:
```

Comment:
Do we do the `assert auth.nonce < 2**64` or else the tx is invalid? Probably, and I just haven't come across it yet.

Reply:
We don't do this explicitly but is implied in the fact that auth.nonce is U64.